app/views/reports/itt-new-starter-data-sign-off.html

```diff
@@ -19,37 +19,36 @@
       <h1 class="govuk-heading-l">{{pageHeading}}</h1>
 
       <p class="govuk-body">
-        This export will show the trainee data you’ve provided to the DfE for trainees starting their initial teacher training (ITT) in the {{academicYear}} academic year. We’re calling these trainees ‘new starters’.
+        This export will show the new trainee data you’ve provided to the DfE for trainees who’ve started their initial teacher training (ITT) in the 2022 to 2023 academic year, and have a trainee start date on or before the second Wednesday of October (sometimes called the census date).
```

Comment:
```suggestion
        This export will show the new trainee data you’ve provided to the DfE for trainees who’ve started their initial teacher training (ITT) in the {{ academicYear }} academic year, and have a trainee start date on or before the second Wednesday of October {{ academicYearShort }} (sometimes called the census date).
```

Adding the year to the end ... I think we should be explicit.